src/main/java/com/swirlds/streamloader/processing/AllowanceProcessingBlock.java

```diff
@@ -0,0 +1,144 @@
+				}
+			}
+
+			// check for granting new allowance(s) ...
+			if (transactionBody.hasCryptoApproveAllowance()) {
+				CryptoApproveAllowanceTransactionBody allowance = transactionBody.getCryptoApproveAllowance();
+				// first check for hbar allowances
+				for (int j = 0; j < allowance.getCryptoAllowancesCount(); j++) {
+					CryptoAllowance cryptoAllowance = allowance.getCryptoAllowances(j);
+					AllowanceChange newAllowance = new AllowanceChange(consensusTimestamp, 
+							cryptoAllowance.getOwner().getAccountNum(),
+							cryptoAllowance.getSpender().getAccountNum(), "hbar", cryptoAllowance.getAmount(), true,
+							BalanceChange.HBAR_TOKEN_TYPE);
+					allowanceChanges.add(newAllowance);
+				}
+				// next, check for token allowances
+				for (int j = 0; j < allowance.getTokenAllowancesCount(); j++) {
+					TokenAllowance tokenAllowance = allowance.getTokenAllowances(j);
+					AllowanceChange newAllowance = new AllowanceChange(consensusTimestamp, 
+							tokenAllowance.getOwner().getAccountNum(),
+							tokenAllowance.getSpender().getAccountNum(), "token", tokenAllowance.getAmount(), true,
+							tokenAllowance.getTokenId().getTokenNum());
+					allowanceChanges.add(newAllowance);
+				}
+				// next, check for nft allowances
+				for (int j = 0; j < allowance.getNftAllowancesCount(); j++) {
+					NftAllowance nftAllowance = allowance.getNftAllowances(j);
+					AllowanceChange newAllowance = new AllowanceChange(consensusTimestamp, 
+							nftAllowance.getOwner().getAccountNum(),
+							nftAllowance.getSpender().getAccountNum(), "nft", 1L, true,
+							nftAllowance.getTokenId().getTokenNum());
+					allowanceChanges.add(newAllowance);
+				}
+			}
+
+			// also check for removing allowance(s) ...
+			if (transactionBody.hasCryptoDeleteAllowance()) {
+				CryptoDeleteAllowanceTransactionBody allowance = transactionBody.getCryptoDeleteAllowance();
+				// no hbar or token remove allowances, only NFTs.
+				for (int j = 0; j < allowance.getNftAllowancesCount(); j++) {
+					NftRemoveAllowance nftAllowance = allowance.getNftAllowances(j);
+					AllowanceChange newAllowance = new AllowanceChange(consensusTimestamp, 
+							nftAllowance.getOwner().getAccountNum(),
+							nftAllowance.getOwner().getAccountNum(), "nft", 1L, false,
+							nftAllowance.getTokenId().getTokenNum());
+					allowanceChanges.add(newAllowance);
+				}
+			}
+		}
+		final List<GenericRecord> records = new ArrayList<>();
+		// create json for allowance changes
+		for (AllowanceChange allowanceChange : allowanceChanges) {
+			records.add(new GenericRecordBuilder(ALLOWANCE_AVRO_SCHEMA)
+					.set("consensus_timestamp", allowanceChange.consensusTimeStamp())
+					.set("owner", allowanceChange.owner())
+					.set("spender", allowanceChange.spender())
+					.set("allowance_type", allowanceChange.allowanceType())
+					.set("amount", allowanceChange.amount())
+					.set("is_approval", allowanceChange.isApproval())
+					.set("token_id", allowanceChange.tokenId())
```

Comment:
Can we just add an empty "errata" field as of now?

Reply:
I was going to do this, but then you told me you were going to drop the column.  Let me know if you change your mind (and want an empty "errata" field) -- that would be an easy change.

Reply:
I had added the comment before our discussion.:)